roles/openshift_metrics/vars/main.yaml

```diff
@@ -1,4 +1,20 @@
+---
+#
+# These vars are generally considered private and not expected to be altered
+# by end users
+#
```

Comment:
The problem with putting variables here is [they can't be overridden][https://docs.ansible.com/ansible/playbooks_roles.html#role-default-variables].